### PR TITLE
add get in touch and contact form sample

### PR DESCRIPTION
**Pull request by Besufekad-HAZ**
-Add get in touch header text
-Add contact form name and address

### CODE REVIEW

**Comment by Besufekad-HAZ** [comment]
-add the contact form
-add the remaining footer section including email and social icons.
-make the form check and validate user inputs(reg expression, upper case and lower case letters, email format, etc...)
-add submit button to submit the form to form-spree and reply with a success message.

File: index.html [hunk 1]
```diff
@@ -231,5 +231,97 @@ <h3 class="proj__title">Skills</h3>
...
+              id="msg"
+              placeholder="Enter your message here..."
+              maxlength="500"
+              required
+            ></textarea>
+            <li><p class="success_message" id="error_message"></p></li>
+            <li class="align_center">
+              <button class="collab__btn" id="fot" type="submit">
+                Get in touch
+              </button>
```
Comment (Ibizugbe) [inline index.html:289]: I can see you implemented a nice animation when your button is active👍👍 but  you haven't implemented the different interactions for the button as stated in the design, kindly see the image below and implement them🤗
template3___States

Comment (Ibizugbe) [inline index.html:289]: I think your button is looking different from the given design🤗, kindly fix it as it is a requirement for this milestone that you stick to the design🤗

your application
your_get_______Touch

the design
get______________touch

File: index.html [hunk 2]
```diff
@@ -231,5 +231,97 @@ <h3 class="proj__title">Skills</h3>
         </article>
       </div>
     </main>
+
+    <footer class="contact__form">
+      <div class="contact__title">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
```
Comment (Ibizugbe) [inline index.html:240]: Kindly stick to the font color given in the design, the color of your test is looking different from the design🤗
your font
your_____font

design font
design_____font

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Besufekad-HAZ 👋, 

68747470733a2f2f6d65646961342e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!🤗

## Highlights✅
- Animation looks good on the button👍
- You have used the right github flow method👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Besufekad-HAZ) [reply index.html:240]: Thanks, for the feedback.

**Review by IngDavidTM — APPROVED** [review]
Hi @Besufekad-HAZ  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**